nucypher/utilities/prometheus/collector.py

```diff
@@ -226,9 +190,6 @@ def initialize(self, metrics_prefix: str, registry: CollectorRegistry) -> None:
             "worker_eth_balance_gauge": Gauge(f'{metrics_prefix}_worker_eth_balance',
                                               'Operator Ethereum balance',
                                               registry=registry),
-            "worker_token_balance_gauge": Gauge(f'{metrics_prefix}_worker_token_balance',
```

Comment:
Since this is removed from initialization, it also needs to be removed from collection logic - L239.